week-1-practise/level-3/1-piping.js

```diff
@@ -7,10 +7,14 @@
   - format
   - assign to the variable `result`
 
-  As a learning exercise (you wouldn't do this normally), do all of the above in 1 line of code 
+  As a learning exercise (you wouldn't do this normally), do all of the above in 1 line of code
 */
 
 var startingValue = 5;
+var number = 2;
+var adding = 10;
+
+const result = multiply(startingValue, number) + add(startingValue, startingValue);
```

Comment:
technically, you need to delete lines 14 and 15 to meet the requirements! 😸 

You are nearly there, see if you can delete those lines and make the result the same...